### PR TITLE
Making user event table similar to other event tables

### LINKED ISSUE #2680: User section events table is not up-to-date with other tables of such type
**Describe the bug**

If you see the events table of a specific user , The event table shown over there is not up to date with other.

**To Reproduce**

Steps to reproduce the behaviour:

1. Go to 'Admin/Users/Events'

**Expected behaviour**

Table should be like other events table

**Screenshots**

#### Current
Screenshot 2019-04-16 at 1 32 47 PM
Expected
Screenshot 2019-04-16 at 1 47 04 PM

**Desktop (please complete the following information):**

 - OS: MacOS Mojave
 - Browser: Safari
 - Version: 23.0.3

**Additional context**

Would like to work on it

### PR DESCRIPTION
**Pull request by kushthedude**
#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

#### Short description of what this resolves:

Fixes #2680 
#### Changes proposed in this pull request:

- Made the user events table similar to other event tables.
- Made some UI enhancements in `Draft` and `Past` event section of users

Fixes #2680

### CODE REVIEW

**Comment by kushthedude** [comment]
@CosmicCoder96 @pradeepgangwar @ritikamotwani Please review it .

**Comment by pradeepgangwar** [comment]
@kushthedude can you attach some screenshots for quick review.

**Comment by kushthedude** [comment]
Screenshot 2019-04-16 at 5 08 58 PM
Screenshot 2019-04-16 at 5 09 04 PM
@pradeepgangwar Please review

File: app/templates/admin/users/view/events.hbs [hunk 1]
```diff
@@ -1,20 +1,21 @@
...
-          {{/link-to}}
-          {{#link-to 'admin.users.view.events.list' 'draft' class='item'}}
-            {{t 'Draft'}}
-          {{/link-to}}
-          {{#link-to 'admin.users.view.events.list' 'past' class='item'}}
-            {{t 'Past'}}
-          {{/link-to}}
-        {{/tabbed-navigation}}
-      </div>
+
```
Comment (pradeepgangwar) [inline app/templates/admin/users/view/events.hbs]: Remove this extra empty line.

**Comment by kushthedude** [comment]
Did the change @pradeepgangwar

**Comment by kushthedude** [comment]
@ritikamotwani changed the message

**Pull request merged**